onnx/defs/math/defs.cc

```diff
@@ -2594,4 +2594,157 @@ ONNX_OPERATOR_SET_SCHEMA(
             propagateShapeFromInputToOutput(ctx, 0, 1);
           }
         }));
+
+static int64_t fft_signal_ndim_default = 1;
+
+static const char* FFT_ver13_doc = R"DOC(
+Fast Fourier Transform.
+
+This implementation suppose the DFT (Discrete Fourier Transform) defined by
+```
+A_k =  \sum_{m=0}^{n-1} a_m \exp\left\{-2\pi i{mk \over n}\right\}
+\qquad k = 0,\ldots,n-1.
+```
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    FFT,
+    13,
+    OpSchema()
+        .SetDoc(FFT_ver13_doc)
+        .Attr(
+            "signal_ndim",
```

Comment:
Specifying the number-of-dimensions as an attribute is uncommon for ONNX ops. Will this add any value, or can we omit it? It is okay, however, for the spec to restrict the number of dimensions (to 1, 2, or 3) if that is useful. But